docs/features/djangorestframework.md

```diff
@@ -36,15 +36,63 @@ validation.
 
 ````
 # models.py
+from ai_django_core.models import CommonInfo
+
 class MyOwnershipRelevantModel(CommonInfo):
     ...
 
 # serializers.py
+from ai_django_core.drf.serializers import CommonInfoSerializer
+
 class MyOwnershipRelevantModelSerializer(CommonInfoSerializer)
     class Meta:
         model = MyOwnershipRelevantModel
+        
+        # use __all__ to show all fields including the common ones
+        fields = '__all__' 
+
+        # or access them directly with 
+        # 'created_at', 'lastmodified_at',
+        # 'created_by', 'lastmodified_by'
+        # for instance
+        fields = ['id', 'name', 'created_at', 'lastmodified_at']
```

Comment:
@jqueguiner please comment this out so if somebody just copy& paste's the code it should work (as much as possible)